packages/core-packages-documentation-generator/src/index.ts

```diff
@@ -0,0 +1,21 @@
+import { ParameterType, PluginHost } from "typedoc/dist/lib/utils";
+
+import { SdkIndexLinkClientPlugin } from "./sdk-index-link-client";
+
+/**
+ *
+ * @param pluginHost An instance of PluginHost.
+ */
+module.exports = function load(pluginHost: PluginHost) {
+  const application = pluginHost.owner;
+  // Core packages doc generator plugins
```

Comment:
nit: period and a newline.
```suggestion

  // Core packages doc generator plugins.
```